cmd/cloud-controller-manager/main.go

```diff
@@ -75,17 +75,17 @@ func main() {
 
 	cloud, err := cloudprovider.InitCloudProvider(cloudProviderName, c.ComponentConfig.KubeCloudShared.CloudProvider.CloudConfigFile)
 	if err != nil {
-		klog.Fatalf("Cloud provider could not be initialized: %v", err)
+		klog.ErrorS("Cloud provider could not be initialized: %v", err)
```

Comment:
Please take a look at https://kubernetes.io/blog/2020/09/04/kubernetes-1-19-introducing-structured-logs/ fo details on using structured logging. Also I believe err should be the first parameter to ErrorS.